get_http_resposne_status.rb

```
@@ -0,0 +1,15 @@
+
+OPTIONS = {}
+OptionParser.new do |opt|
+  opt.on('-u url', '--url', String) {|v| OPTIONS[:url] = v}
+  opt.parse(ARGV)
+end
+
+raise OptionParser::MissingArgument, 'NoURLFoundError' unless OPTIONS[:url]
+
+res = URI.parse(OPTIONS[:url])
+_  = res
+binding.pry
```

rubocop (1.69.1)

[warning] Remove debugger entry point `binding.pry`.

(Lint/Debugger)